app/(tabs)/(touren)/createManualStage.tsx

```diff
@@ -0,0 +1,200 @@
+  //switches title from plain text to the input field
+  const [titleBeingChanged, setTitleBeingChanged] = useState(false);
+  const [stageTitle, setStageTitle] = useState("Etappe"); ///the name of the title
+  const router = useRouter();
+  // compass input
+  const [startLat, setStartLat] = useState("");
+  const [startLon, setStartLon] = useState("");
+  const [startDate, setStartDate] = useState(new Date());
+
+  const [endLat, setEndLat] = useState("");
+  const [endLon, setEndLon] = useState("");
+  const [endDate, setEndDate] = useState(new Date());
+
+  ////////////address input
+
+  ////////////on the map input
+
+  const titleInput = useMemo(
+    () => (
+      <Layout style={styles.headerInput}>
+        <Input
+          value={stageTitle}
+          onChangeText={(nextValue) => setStageTitle(nextValue)}
+          maxLength={20}
+        />
+      </Layout>
+    ),
+    [stageTitle],
+  );
+
+  //switching between done and edit
+
+  const changingTitleButton = useMemo(
+    () => (
+      <Button
+        appearance="ghost"
+        onPress={() => setTitleBeingChanged((prevState) => !prevState)}
+      >
+        {titleBeingChanged ? (
+          <FontAwesomeIcon icon="check" size={25} />
+        ) : (
+          <FontAwesomeIcon icon="edit" size={25} />
+        )}
+      </Button>
+    ),
+    [titleBeingChanged],
+  );
+
+  // Update header input ot text based on if title or edit state changes
+  useEffect(() => {
+    navigation.setOptions({
+      headerTitle: () =>
+        titleBeingChanged ? (
+          titleInput
+        ) : (
+          <Text category="h4">{stageTitle}</Text>
+        ), // Switch between input and title
+      headerRight: () => changingTitleButton, // Set the button on the right
+      headerTitleAlign: "center", // Keep the title centered
+    });
+  }, [
+    navigation,
+    titleInput,
+    changingTitleButton,
+    titleBeingChanged,
+    stageTitle,
+  ]);
+
+  const startCoordInput = (
+    <CoordinateInput
+      lat={startLat}
+      lon={startLon}
+      setLat={setStartLat}
+      setLon={setStartLon}
+      date={startDate}
+      setDate={setStartDate}
+    />
+  );
+
+  const endCoordInput = (
+    <CoordinateInput
+      lat={endLat}
+      lon={endLon}
+      setLat={setEndLat}
+      setLon={setEndLon}
+      date={endDate}
+      setDate={setEndDate}
+    />
+  );
+
+  const onSubmitEtappe = async () => {
+    try {
+      await createManualStageFn(
+        stageTitle,
+        startLat + ", " + startLon,
+        endLat + ", " + endLon,
+        startDate,
+        endDate,
+        tourId,
+      );
+      router.back();
+    } catch (err) {
+      if (err instanceof Error) {
+        Alert.alert("Error", err.message);
+      } else {
+        Alert.alert("Unknown Error", "An unexpected error occurred.");
+      }
+    }
+  };
+
+  return (
+    <Layout style={styles.layout} level="2">
+      <ButtonGroup
+        style={{
+          borderRadius: 10,
+        }}
+      >
```

Comment:
```suggestion
      <ButtonGroup>
```

style existiert nicht in ButtonGroup